PluginCode/src/main/java/PostStartupActivity.java

```diff
@@ -34,9 +39,21 @@ public void selectionChanged(@NotNull FileEditorManagerEvent event) {
                 } catch (IOException e) {
                     e.printStackTrace();
                 }
+            } // public void selectionChanged
+        });
 
+        connection.subscribe(VirtualFileManager.VFS_CHANGES, new BulkFileListener() {
+            @Override
+            public void after(@NotNull List<? extends VFileEvent> events) {
+                // handle the events
```

Comment:
which events? Can you be specific so it's easy in the future to understand the code?